File: src/printing/printing_stdout_1.py
if __name__ == "__main__":
    value_1 = "Hello"
    value_2 = "World"

    # Simple write to stdout,
    # with new line, and space between printed objects.
    print(value_1, value_2)

    # Write to stdout (change space between objects to "_").
    print(value_1, value_2, sep="_")

    # Simple write to stdout (without new line)
    print(value_1, value_2, end="")

File: src/printing/printing_stdout_1.rs
fn main() {
    // Rust's compiler automatically determined `&str` type,
    // so no need to declare it.
    let value_1 = "Hello";
    let value_2 = "World";

    // Simple write to stdout,
    // with new line, and space between printed objects.
    println!("{} {}", value_1, value_2);
    // Write to stdout (change space between objects to "_").
    println!("{}_{}", value_1, value_2);
    // Simple write to stdout (without new line).
    print!("{} {}", value_1, value_2);
}
